### PR TITLE
add cases for MAC addresses in EVE-JSON

### PR DESCRIPTION
**Pull request by satta**
This relates to a PR in the main suricata repo (https://github.com/OISF/suricata/pull/4660). It tests the correctness of output for flow and packet events when facing multiple or single MAC addresses per source/destination.

### CODE REVIEW

File: tests/mac-eve-multiple-disabled/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
```
Comment (norg) [inline tests/mac-eve-multiple-disabled/suricata.yaml:7]: maybe remove the comment part as it's not necessary :)

Comment (satta) [reply tests/mac-eve-multiple-disabled/suricata.yaml:7]: Sure, the cleaner the better!

File: tests/mac-eve-multiple/suricata.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
```
[the remark this answers is not in the document]
Comment (satta) [reply tests/mac-eve-multiple/suricata.yaml:7]: OK. Next PR :)

**Comment by satta** [comment]
New PR: #237

**Pull request closed without merging**